src/ServiceManagement/ExpressRoute/ExpressRouteManagement/Properties/AssemblyInfo.cs

```diff
@@ -20,7 +20,7 @@
 [assembly: AssemblyDescription("Provides express route management capabilities to developers. Create, delete, list, retrieve dedicated circuits, bgp peerings, dedicated circuit links, and more.")]
 
 [assembly: AssemblyVersion("0.9.0.0")]
-[assembly: AssemblyFileVersion("0.18.14.0")]
+[assembly: AssemblyFileVersion("0.19.14.0")]
```

Comment:
@dihan0604 if you bump up minor version, patch version should be 0. should be 0.19.0